### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by WeihongM** [kanishkg/textboxes#1]
Hello, 
May I ask have you finished your code of textbox based on pytorch implementation? In your repo, it seems that you have finished a part.
Hope for your help, thanks.

**Comment by kanishkg** [comment]
Hey,
I was working on this for fun, but got distracted by other projects midway. I think the model part is complete albeit untested. I'd be happy to help.

**Comment by WeihongM** [comment]
okay，I will refer to your code. Do you have interest to finish your code completely?

**Comment by kanishkg** [comment]
No, at least not in the near future. Please feel free to fork and complete it if you wish to.
